crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -413,52 +431,160 @@ impl<'src> Parser<'src> {
         self.bump(TokenKind::From);
 
         let mut module = None;
-        let mut level = if self.eat(TokenKind::Ellipsis) { 3 } else { 0 };
+        let mut leading_dots = if self.eat(TokenKind::Ellipsis) { 3 } else { 0 };
```

Comment:
Can we change this to `0` considering that the first ellipsis will always be eaten by the while loop?
```suggestion
        let mut leading_dots = 0;
```